app/component/route/route.scss

```diff
@@ -211,12 +213,17 @@ div.route-tabs {
 }
 
 .nearby-routes {
-  overflow-y: auto;
+  overflow: hidden;
```

Comment:
There was a double overflow (the whole tab, and the list below) which was not good. So it's better to just hide the one for the whole tab contents. There's actually not going to be any need to scroll when the extra scrollbar is gone.